diagnostic.md

```diff
@@ -44,9 +56,20 @@ How would we reference each of the following categories of elements?
 -   Only those elements belonging to both the `important` and `red` _classes_
 -   All `p` elements whose parent elements are `div`s
 
-<!-- your answer starts here -->
+```CSS
+.big {
 
-<!-- your answer ends here -->
+}
+#contentPane {
+
+}
+.important, .red {
```

Comment:
This will select all elements that have at least one of the two classes, ```important``` or ```red```.  To select elements that have both ```important``` and ```red```, use ```.important.red```.